echopype/convert/set_groups_ad2cp.py

```diff
@@ -417,21 +414,55 @@ def set_beam(self) -> List[xr.Dataset]:
             if "ping_time" not in ds.dims:
                 beam_groups[i] = ds.expand_dims(dim="ping_time")
 
+        # remove time1 from beam groups
+        for i, ds in enumerate(beam_groups):
+            beam_groups[i] = ds.sel(time1=ds["ping_time"])
+
         return [set_encodings(ds) for ds in beam_groups]
 
     def set_vendor(self) -> xr.Dataset:
         ds = self._make_dataset(
             {
                 "version": "data_record_version",
+                "pressure_sensor_valid": "pressure_sensor_valid",
+                "temperature_sensor_valid": "temperature_sensor_valid",
+                "compass_sensor_valid": "compass_sensor_valid",
+                "tilt_sensor_valid": "tilt_sensor_valid",
+                "velocity_data_included": "velocity_data_included",
+                "amplitude_data_included": "amplitude_data_included",
+                "correlation_data_included": "correlation_data_included",
+                "altimeter_data_included": "altimeter_data_included",
+                "altimeter_raw_data_included": "altimeter_raw_data_included",
+                "ast_data_included": "ast_data_included",
+                "echosounder_data_included": "echosounder_data_included",
+                "ahrs_data_included": "ahrs_data_included",
+                "percentage_good_data_included": "percentage_good_data_included",
+                "std_dev_data_included": "std_dev_data_included",
+                "distance_data_included": "distance_data_included",
+                "figure_of_merit_data_included": "figure_of_merit_data_included",
                 "error": "error",
-                "status": "status",
                 "status0": "status0",
```

Comment:
here you kept `status0` and removed `status` -- is this intentional? I think it'll be fine to keep both of them, since the comments use that reference.

Reply:
It's still there a few lines below

Reply:
Oops I had missed that. Thanks!